app/src/main/java/org/oppia/app/options/ReadingTextSizeFragmentPresenter.kt

```diff
@@ -114,11 +119,12 @@ class ReadingTextSizeFragmentPresenter @Inject constructor(private val fragment:
   }
 
   fun getReadingTextSizeInFloat(readingTextSize: ReadingTextSize): Float {
+    val defaultReadingTextSizeInFloat = 18f
```

Comment:
suggest defining this in `dimens.xml` file.

Reply:
sure

Reply:
@rt4914 So, that should be in sp, right?

Reply:
@rt4914 
``` val defaultReadingTextSizeInFloat = fragment.context?.resources?.getDimension(R.dimen.defaultReadingTextSize)```

Is this correct or is there any other better way to use it here?

Reply:
Yeah you can do this but make sure to use `default_reading_text_size` as format